Singletons/ItemStash.gd

```diff
@@ -43,3 +44,11 @@ func get_item_container() -> ItemContainer:
 
 func _on_item_container_items_changed():
 	items_changed.emit()
+
+
+func _on_game_mode_was_chosen():
```

Comment:
Add a comment explaining that these items are added so that player can practice adding items to towers during tutorial.

Reply:
Done